indico/modules/events/timetable/client/js/Timetable.tsx

```diff
@@ -80,15 +83,34 @@ function GlobalEvents() {
         dispatch(actions.redoChange());
       } else if (e.key === 'Escape') {
         // deselect selected entry
-        if (selectedId) {
+        if (selectedEntry) {
           dispatch(actions.deselectEntry());
         }
+      } else if (e.key === 'Delete') {
+        if (selectedEntry) {
+          switch (selectedEntry.type) {
+            case EntryType.Break: {
+              dispatch(actions.deleteBreak(selectedEntry, eventId));
+              break;
+            }
+            case EntryType.SessionBlock: {
+              dispatch(actions.deleteBlock(selectedEntry, eventId));
+              break;
+            }
+            case EntryType.Contribution: {
+              dispatch(actions.unscheduleEntry(selectedEntry, eventId));
+              break;
+            }
+            default:
+              break;
+          }
```

Comment:
I'm wondering if this is a little verbose for what it is. We do something like this in the TimetableManageModal

```ts
{
          [EntryType.Contribution]: _handleCreateContribution,
          [EntryType.SessionBlock]: _handleCreateSessionBlock,
          [EntryType.Break]: _handleCreateBreak,
  }[type];
  ```
  
  You can substitute the functions in there with the action functions, as you do dispatch and pass the same props to all of them anyway